docs/content/guide/di.ngdoc

```diff
@@ -14,23 +14,30 @@ and providing them to other components as requested.
 
 ## Using Dependency Injection
 
-DI is pervasive throughout AngularJS. You can use it when defining components or when providing `run`
-and `config` blocks for a module.
-
-- Components such as services, directives, filters, and animations are defined by an injectable
-factory method or constructor function. These components can be injected with "service" and "value"
-components as dependencies.
-
-- Controllers are defined by a constructor function, which can be injected with any of the "service"
-and "value" components as dependencies, but they can also be provided with special dependencies. See
-{@link di#controllers Controllers} below for a list of these special dependencies.
-
-- The `run` method accepts a function, which can be injected with "service", "value" and "constant"
-components as dependencies. Note that you cannot inject "providers" into `run` blocks.
-
-- The `config` method accepts a function, which can be injected with "provider" and "constant"
-components as dependencies. Note that you cannot inject "service" or "value" components into
-configuration.
+Dependency Injection is pervasive throughout AngularJS. You can use it when defining components
+or when providing `run` and `config` blocks for a module.
+
+- {@link angular.Module#service Services}, {@link angular.Module#directive directives},
+{@link angular.Module#filter filters}, and {@link angular.Module#animation animations} are
+defined by an injectable factory method or constructor function, and can be injected with
+"service" and "value" as dependencies.
+
+- {@link ng.$controller Controllers} are defined by a constructor function, which can be injected
+with any of the "service" and "value" as dependencies, but they can also be provided with
+special dependencies. See {@link di#controllers Controllers} below for a list of these
+special dependencies.
+
+- The {@link angular.Module#run `run`} method accepts a function, which can be injected with
+"service", "value" and "constant" as dependencies. Note that you cannot inject "providers"
+into `run` blocks.
+
+- The {@link angular.Module#config `config`} method accepts a function, which can be injected with
+"provider" and "constant" as dependencies. Note that you cannot inject "service" or
+"value" components into configuration.
+
+- The {@link angular.Module#provider `provider`} method can only be injected with other "provider"
```

Comment:
For consistency, I would just say `"providers"` rather than `"provder" injectables`, unless you change the rest to use the word injectables too.